style.css

```diff
@@ -0,0 +1,106 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@300&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.nav {
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 5vh;
+  padding: 0 5%;
+}
+
+.logo {
+  font-family: "Roboto", serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.125rem;
+  line-height: 1.5rem;
+  color: #fff;
+}
+
+.mobileMenu {
+  color: #fff;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  height: 95vh;
+  width: 100%;
+  background-color: #1c1a19;
+  color: #ff6b00;
+}
+
+.headline > h1 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 3.5rem;
+  line-height: 4rem;
+  letter-spacing: 0.023rem;
+  color: #ff6b00;
+  margin-top: 7.5rem;
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
+  margin-bottom: 0.75rem;
+}
+
+.headline > h2 {
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
+  font-family: "Roboto", serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  margin-bottom: 0.75rem;
+  color: #fff5e1;
+}
+
+.headline > p {
+  font-family: "Poppins", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
+  margin-bottom: 0.75rem;
+  color: #fff;
+}
+
+.socials {
+  display: flex;
+  margin-left: 1.5rem;
+  margin-right: 9.438rem;
+  width: min-content;
+}
+
+.socialLink {
+  width: 2.5rem;
+  height: 2.5rem;
+}
+
+.bgImage {
+  position: absolute;
+  top: 45.5%;
+  right: 0%;
+}
+
+@media (max-width: 375px) {
+  .bgImage {
+    display: none;
+  }
+}
```

Comment:
- [ ] Kindly make sure to show your background image even on the mobile versions.

   - ### YOUR DESIGN
   
      ![image](https://user-images.githubusercontent.com/87197412/219029506-01b751c9-9a47-4fa2-abf5-c069259cf7a2.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/219029671-3dd60e4d-fb0a-4547-a4a5-f10ca7fc2090.png)